INCHI-1-SRC/INCHI_API/demos/inchi_main/src/CMakeLists.txt

```
@@ -0,0 +1,99 @@
+cmake_minimum_required(VERSION 3.21 FATAL_ERROR)
+
+project(InChIMain_API VERSION 1.7)
+
+add_library(inchimain_compiler_flags INTERFACE)
+target_compile_features(inchimain_compiler_flags INTERFACE c_std_11)
+
+set(gcc_like_cxx "$<COMPILE_LANG_AND_ID:C,ARMClang,AppleClang,Clang,GNU,LCC>")
+set(msvc_cxx "$<COMPILE_LANG_AND_ID:C,MSVC>")
+target_compile_options(inchimain_compiler_flags INTERFACE
+	"$<${gcc_like_cxx}:$<BUILD_INTERFACE:-ansi;-g;-O1;-c;-fno-strict-aliasing;-Wno-all>>"

```

`-ansi` overwrites `c_std_11`, see #115
```suggestion
	"$<${gcc_like_cxx}:$<BUILD_INTERFACE:-g;-O1;-c;-fno-strict-aliasing;-Wno-all>>"

```